src/debugger/sdfg_python_debugger.ts

```diff
@@ -1,7 +1,8 @@
-// Copyright 2020-2021 ETH Zurich and the DaCe-VSCode authors.
+// Copyright 2020-2022 ETH Zurich and the DaCe-VSCode authors.
 // All rights reserved.
 
 import * as vscode from 'vscode';
+import { executeTrusted } from '../utils/utils';
 import { SdfgPythonDebugSession } from './sdfg_python_debug_session';
 import { FileAccessor } from './sdfg_python_runtime';
```

Comment:
Please do not run the daemon at all if untrusted. We should not do anything outside of the webview or anything that allows it access to the filesystem.

Reply:
The dace daemon can now only be started with a trusted workspace. If the workspace is set to trusted, and the user has already attempted to start the daemon, this will now automatically start it upon trusting the workspace, refreshing transformations etc. once the daemon has started.